### PR TITLE
Area mark

### PR DESCRIPTION
**Pull request by deathbearbrown**
Add the vega Area Mark to Lyra2.
#### This PR includes:
- [x] Tests
- [x] functional comments
- [x] high level description of any new components
### Steps to functionally test this:

Ok, this mark is weird.  See: #289

The Mark exists on the stage and you can add data to the axis and change interpolation and such, but it makes a fairly ugly graph without having the index on the other axis.

### CODE REVIEW

File: src/js/components/Inspector.jsx [hunk 1]
```diff
@@ -40,4 +40,4 @@ module.exports = Inspector;
 Inspector.Line = require('./inspectors/Line');
 Inspector.Rect = require('./inspectors/Rect');
 Inspector.Symbol = require('./inspectors/Symbol');
-Inspector.Text = require('./inspectors/Text');
+Inspector.Area = require('./inspectors/Area');
```
Comment (kadamwhite) [inline src/js/components/Inspector.jsx:44]: You seem to have clobbered Text :)

**Comment by kadamwhite** [comment]
Once the inspector for Text marks has been restored, I think I'm +1, but I need some suggestions about which properties to drag onto which channel to get something to render

File: src/js/components/Inspector.jsx (hunk 1 above)
Comment (deathbearbrown) [inline src/js/components/Inspector.jsx:44]: I'm a rebasing monster

**Comment by deathbearbrown** [comment]
screen shot 2016-03-11 at 9 41 02 am

Ok I pushed a fix.  @arvind this still looks crazy to me. Can you check this too?

**Comment by kadamwhite** [comment]
image
I made a dapper bow tie for Cluster! And this is good to go; mergin'. Nice work, @deathbearbrown

**Pull request merged**